general-remarks.md

```diff
@@ -100,14 +100,18 @@ Or
 | Property | Type |  | Description |
 | --- | --- | --- | --- |
 | `success` | `bool` | required | `true` or `false` determining result of operation. |
-| `error` | [`Error`](general-remarks.md#error) | optional | In case of `"success": false`, this property holds information of the error that happened. |
+| ~~`error`~~ | ~~[`Error`](general-remarks.md#error)~~ | ~~optional~~ | ~~In case of `"success": false`, this property holds information of the error that happened.~~ |
```

Comment:
Do you have followup cleanup tasks for all such places?

Reply:
From the far past we dont now i will connect it with the deprecation issue.

Reply:
The you should figure it out and remove all the things that are currently crossed out and have been like this for long time. You should also remove it from the API completely.